src/lxc/initutils.c

```diff
@@ -139,11 +139,11 @@ const char *lxc_global_config_value(const char *option_name)
 	}
 
 	const char * const (*ptr)[2];
-	size_t i;
+	size_t option;
```

Comment:
Kinda bike-shedding, but 'option' next to 'options' (both of which can be safely incremented) also seems scary. 'opt' maybe?  Of course the best solution would be to break this function up, but I'm not asking you to do this as part of this PR.